### PR TITLE
Don't set homepage template to `null`, if theme.yml is not present.

### PR DESCRIPTION
**Pull request by bobdenotter**

### CODE REVIEW

File: src/TemplateChooser.php [hunk 1]
```diff
@@ -32,7 +32,9 @@ public function homepage($content)
         $templates = [];
 
         // First candidate: Theme-specific config.yml file.
-        $templates[] = $this->config->get('theme/homepage_template');
+        if ($this->config->get('theme/homepage_template')) {
```
Comment (CarsonF) [inline src/TemplateChooser.php]: Could save the second look up by saving the result here.

File: src/TemplateChooser.php [hunk 2]
```diff
@@ -32,7 +32,9 @@ public function homepage($content)
...
+            $templates[] = $this->config->get('theme/homepage_template');
+        }
 
         // Second candidate: Global config.yml file.
         $templates[] = $this->config->get('general/homepage_template');
```
Comment (CarsonF) [inline src/TemplateChooser.php]: Will this one always exist?

Comment (bobdenotter) [inline src/TemplateChooser.php]: It should, unless somebody axplicitly deletes it from their config.yml

File: src/TemplateChooser.php (hunk 1 above)
Comment (bobdenotter) [inline src/TemplateChooser.php]: I know, but I made it in line with the dozen or so other similar statements in that file. If we change this one, we should change all of them to keep it consistent.

**Pull request merged**